evaluationquestions.txt

```diff
@@ -0,0 +1,91 @@
+////////////////    part A      ////////////////
+initialize an array with 2 values in it (apples and oranges)
+const fruit = ["apples", "oranges"];
+
+what does array.map do? and what is returns?
+map.array method creates a new array by iterating over the array and calling function on every element of array, it return the same array length with the ability to change array elements
+fruit.map(element => {
+    element * 3.5;
+});
+
+what does array.filter do? and what is returns?
+array.filter method creates an array filter filled with the array filtered elements, it return the filtered element of the array with the ability to change these returned elements
```

Comment:
same as above, let's not forget to return

Reply:
yes, noted

Reply:
done